.github/workflows/uploadDockerImage.yml

```diff
@@ -3,17 +3,28 @@ name: Upload tag to GitHub
 on:
   push:
     tags:
-      - "*"
+      - '*'
 
 jobs:
   build:
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@master
-      - name: Publish to Registry
-        uses: elgohr/Publish-Docker-Github-Action@master
-        with:
-          name: docker.pkg.github.com/danger/danger-js/release
-          username: ${{ secrets.DOCKER_GITHUB_USERNAME }}
-          password: ${{ secrets.DOCKER_GITHUB_PASSWORD }}
-          registry: docker.pkg.github.com
+    - uses: actions/checkout@v2
+    
+    - name: Export tag version
+      run: echo ::set-env name=VERSION::${GITHUB_REF/refs\/tags\//}
+    
+    - name: Docker Login
+      run: echo $PACKAGES_WRITE_TOKEN | docker login docker.pkg.github.com -u $USERNAME --password-stdin
+      env:
+        PACKAGES_WRITE_TOKEN: ${{ secrets.DOCKER_GITHUB_PASSWORD }}
+        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
+    - name: Docker Build
+      run: docker build -t docker.pkg.github.com/danger/danger-js/danger-danger-js:$VERSION .
+      env:
+        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
+    - name: Deploy
+      run: docker push docker.pkg.github.com/danger/danger-js/danger-danger-js:$VERSION
```

Comment:
Requires auth to download (even public) images. Why not use Docker's registry instead?

Reply:
Yes, I know, I highlighted this in https://github.com/danger/danger-js/pull/810#issuecomment-618001174
But the credentials we have are for GitHub package registry, then I'm fixing the current script, if use or not dockerhub is a separate discussion, and can be done in parallel.